tests/scalers/mssql/mssql_test.go

```diff
@@ -37,6 +41,18 @@ var (
 		mssqlHostname, mssqlDatabase, mssqlPassword)
 	minReplicaCount = 0
 	maxReplicaCount = 5
+
+	// Workload Identity test variables
+	wiTestNamespace     = fmt.Sprintf("%s-ns", wiTestName)
+	wiDeploymentName    = fmt.Sprintf("%s-deployment", wiTestName)
+	wiScaledObjectName  = fmt.Sprintf("%s-so", wiTestName)
+	wiTriggerAuthName   = fmt.Sprintf("%s-ta", wiTestName)
+	wiSecretName        = fmt.Sprintf("%s-secret", wiTestName)
+	wiTriggerSecretName = fmt.Sprintf("%s-ta-secret", wiTestName)
+	azureSQLServerFQDN  = os.Getenv("TF_AZURE_SQL_SERVER_FQDN")
+	azureSQLDBName      = os.Getenv("TF_AZURE_SQL_SERVER_DB_NAME")
+	azureADTenantID     = os.Getenv("TF_AZURE_SP_TENANT")
+	azureClientID       = os.Getenv("AZURE_CLIENT_ID")
```

Comment:
The env var that contains the app id is this:
```suggestion
	azureClientID       = os.Getenv("TF_AZURE_IDENTITY_1_APP_ID")
```

Reply:
I case of necessity (to spin up the database or so), the admin credentials are in these envs:
- TF_AZURE_SQL_SERVER_ADMIN_USERNAME
- TF_AZURE_SQL_SERVER_ADMIN_PASSWORD